device-types/Cisco/MDS-C9148S.yaml

```diff
@@ -0,0 +1,138 @@
+---
+manufacturer: Cisco
+model: MDS 9148S
+slug: ds-c9148s
+part_number: DS-C9148S
```

Comment:
According to [this datasheet](https://www.cisco.com/c/en/us/products/collateral/storage-networking/mds-9148s-16g-multilayer-fabric-switch/datasheet-c78-731523.html) in Table 5, there are 6 different configurations with their own specific part numbers. Which one is this? Please update the file name, part number, model, and slug to include the specific part number that is applicable to this device

Reply:
Updated the Part number to DS-C9148S-D48PSK9 as well as the slug cisco-ds-c9148-d48psk9